context/app/static/js/components/Detail/ProvTabs/ProvTabs.jsx

```diff
@@ -27,30 +36,34 @@ function ProvTabs(props) {
         aria-label="Detail View Tabs"
         TabIndicatorProps={{ style: { backgroundColor: '#9CB965' } }}
       >
-        <StyledTab label="Table" id="tab-0" aria-controls="tabpanel-0" />
-        <StyledTab label="Graph" id="tab-1" aria-controls="tabpanel-1" />
-        {shouldDisplayDag && <StyledTab label="Analysis Details" id="tab-2" aria-controls="tabpanel-2" />}
+        {shouldDisplayTable && <StyledTab label="Table" id="tab-0" aria-controls="tabpanel-0" />}
+        <StyledTab label="Graph" id={`tab-${graphIndex}`} aria-controls={`tabpanel-${graphIndex}`} />
+        {shouldDisplayDag && (
+          <StyledTab label="Analysis Details" id={`tab-${dagIndex}`} aria-controls={`tabpanel-${dagIndex}`} />
+        )}
```

Comment:
I could be mistaken, but I don't think the IDs are required to have any particular structure, so instead of numbering them, just use something like `tab-graph` or `tab-dag`?